### DISCUSSION TITLE
There are automated tests for some items that are marked as "(Manual)"

### DISCUSSION THREAD

**Discussion by austintackaberry** [aquasecurity/kube-bench#906] [Questions and Help] [answered]
**Overview**

**How did you run kube-bench?**
Using job-master.yaml

**What happened?**
There are 2 items (1.2.13 and 1.2.35) that were marked as PASS but the items have (Manual)

```
2021-06-14T22:44:03.222517847Z [PASS] 1.2.13 Ensure that the admission control plugin SecurityContextDeny is set if PodSecurityPolicy is not used (Manual)
```

```
2021-06-14T22:44:03.222734651Z [PASS] 1.2.35 Ensure that the API Server only makes use of Strong Cryptographic Ciphers (Manual)
```

**What did you expect to happen:**

The docs say [here](https://github.com/aquasecurity/kube-bench/blame/main/docs/flags-and-commands.md#L115) that if an item is `Manual`, then it will always be logged as `WARN`. Seems like the item should be renamed to be `(Automated)`

**Environment** 
kube-bench: aquasec/kube-bench:latest
kubernetes version: v1.16.12

**Comment by yoavrotems** [comment, marked as answer]
Hey @austintackaberry, 
Thank you for reaching out and reporting this, but this not a bug.
It just two different manual meanings. 
 `type: "manual"` is a test that for some reason couldn't be automated (for example unpredictable output or organization specific settings). 
Those will always end up with WARN state and could be solved with manual interfering from the user side, if its testing it manually or even writing the test so it will be your needs valid. 
Now about the title "manual" CIS also has their own tool called CAT, and from the last 1.6 cis benchmark I figure that for their needs they decided to change the division from "score/not scored" to "Automated/Manual" probably relate to CAT way of working. 
We stick to CIS benchmarks and there for using those titles which most time indicate on the test manually running or automated but not always.

**Reply by austintackaberry** [reply]
I see, thanks!
